cwl/src/test/scala/wdl4s/cwl/ThreeStepExample.scala

```diff
@@ -0,0 +1,148 @@
+package wdl4s.cwl
+
+import shapeless._
+import syntax.singleton._
+import wdl4s.cwl.CommandLineTool.{Argument, BaseCommand, Inputs, StringOrExpression}
+import wdl4s.cwl.CommandOutputBinding.Glob
+import wdl4s.cwl.WorkflowStep.{Outputs, Run}
+import io.circe.syntax._
+import io.circe.yaml._
+import io.circe.yaml.syntax._
+import org.scalatest._
+
+/*
+ This example calls `ps` , then counts the number of processes that match a pattern input.
+ The output of the ps call is also counted via `wc`.
+ */
+object ThreeStepExample extends App {
```

Comment:
There's no logic to test.  I'd like to avoid testing the JSON encoder i.e. "does X show up in output?"  Not sure if you had something in mind but that's how I see it